### ISSUE TITLE
Newly created Issues in Github not syncing to Linear after adding linear tag

### ISSUE THREAD

**Issue by nickzelei** [calcom/synclinear.com#153]
I created an issue in github: https://github.com/nucleuscloud/neosync/issues/1023

Afterwards, I added the `linear` tag, but the issue never shows up in Linear.

Linear -> Github seems to be working just fine.

**Comment by srini047** [comment]
![image](https://github.com/calcom/synclinear.com/assets/81156510/c5b7b0a5-67cc-4b87-adf0-e9f5c21ff3ef)

From the current website I see, there is only support for Linear->Github sync. But not the other way round like in your case.
So hence needs to be updated manually in Linear if any changes are made at Github

**Comment by tedspare** [comment]
Hey @nickzelei, thanks for sharing. Could you share the error logs from the webhook in your repo > settings > webhooks > SyncLinear > recent deliveries?

**Comment by nickzelei** [comment]
Surely.

I received a 403 for both a `issues.unlabeled` and `issues.labeled`.

For the `issues.labeled` request, here is the response:

Headers:
```
Cache-Control: public, max-age=0, must-revalidate
Content-Length: 69
Content-Type: application/json; charset=utf-8
Date: Sun, 10 Mar 2024 02:43:01 GMT
Etag: "8lx161lqs21x"
Server: Vercel
Strict-Transport-Security: max-age=63072000
X-Matched-Path: /api
X-Vercel-Cache: MISS
X-Vercel-Id: iad1::iad1::x2jb7-1710038580986-54a8eb03d193
```

Body:
```
{"success":false,"message":"GitHub webhook secret doesn't match up."}
```

**Comment by nickzelei** [comment]
Hm, I reconnected the integration and it seems to be working now. Maybe it was that easy.

It's been a few months now since I logged this issue, but I believe I logged it shortly after I had connected the integration.

Just now when I re-connected, I had two webhooks logged in the repo. I have now disconnected the one that was in a bad state (even though linear->github syncs were working).

Here's to hoping that it is continually working. I'll close this since it seems to be fixed!
